EventListener/ViewResponseListener.php

```diff
@@ -27,16 +28,22 @@
  */
 class ViewResponseListener extends TemplateListener
```

Comment:
May be better to just revert this change or the ``ViewResponseListener`` will be dependant of the ``SensioExtraFrameworkBundle``